README.md

```diff
@@ -1,54 +1,191 @@
+[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+[![dev builds | AppVeyor](https://img.shields.io/badge/dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![unique systems emulated | 22](https://img.shields.io/badge/unique_systems_emulated-22-darkgreen.svg?logo=buffer&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/blob/master/README.md#features)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+tl;dr: click the "latest" button above to grab it. The changelog is [here at TASVideos](http://tasvideos.org/Bizhawk/ReleaseHistory.html). **Don't mix versions**, copy `config.ini` to keep settings.
 
-## Download Binaries
+Jump to:
+* [Installing — Windows 7/8.1/10](https://github.com/TASVideos/BizHawk/blob/master/README.md#installing--windows-78110)
+* [Installing — GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#installing--gnulinux-and-macos)
+* [Building/Testing](https://github.com/TASVideos/BizHawk/blob/master/README.md#buildingtesting)
+* [Usage (emulation)](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage-emulation)
+* [Usage (TASing)](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage-tasing)
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-Windows users, don't forget to run the [prereq installer](http://github.com/TASVideos/BizHawk-Prereqs/releases) first!
+## Features
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+The BizHawk common features (across all cores) are:
+* rom format and region detection and a corruption check
+* 10 savestate slots and save/load to file
+* speed control, including frame stepping and rewinding
+* memory view/search/edit in all parts of the emulated HW
+* input recording (making TAS movies)
+* take screenshots and record video
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+System | BH Common | Accuracy† | Debugging | Peripherals
```

Comment:
"BH common" is already mentioned as supported across all cores, no need for it here.

There's no spherical accuracy in vacuum that you can objectively measure. There are test ROMs that we might be passing, then there are games that might work without bugs, but there are always bugs in games, and only a few cores aim to pass all test ROMs. On top of that, there's replicating console behavior that sometimes may not be tested by test ROMs, you just make a movie and tweak the core until the movie syncs on console. Overall, none of that can be measured separately for each core, so this column isn't helpful. Bugs in games are reported regardless.

As I said last time, helpful info would be listing tools supported by cores, and if some tool is supported by all cores, it can be just moved to BH common.

No one tracks peripherals either. If you want, make a table listing all control types each core supports, but it'd be a HUGE table, and it's better to host is at TASVideos.

Reply:
that leaves basically no table left though... I have an alternative idea that might better fit the current state of the project but isn't as visually impressive

Reply:
How is listing supported tools no table?

Reply:
how's that

Reply:
Load a game, pick a core if the given system has several cores, see how many items you see in Tools. Make a table. Also it'd be very nice to list core names per system.

System | Core | Trace Logger | Code Data Logger
--------|------|---------------|-------------------
NES | NESHawk | ✓ | ✓
| | QuickNES | ✓ |

Reply:
I've got the data for the 9 Linux cores already, but a table like that is more verbose than I wanted to put at the top of the readme.

Core names is easy enough, did you mean all of them or just where they overlap?

Reply:
All. If it's too much for the top, move it downwards :D The point is that such a table would be actually relevant and helpful. Linux/Mac support hmm... It makes a lot of sense to tell which core works outside windows. But then we have tools that sometimes don't work. Not sure how to put that. What if we add tiny OS icons before cores and before tools, according to OSs they work on?

Reply:
Maybe it's the TASing section, but since we have 2 types of tables we want to see, OS compatibility for cores and tool availability for cores per OS, why not combine them? Everyone will instantly see what they need to know.

Reply:
holding off on the second table until we know we need it, and I'm really hoping we don't. closing this and adding to desc.